charts/node-analyzer/templates/daemonset-node-analyzer.yaml

```diff
@@ -213,9 +216,10 @@ spec:
             readOnly: true
           - name: tmp-vol
             mountPath: /host/tmp
-{{- end }}
-{{ if not .Values.secure.vulnerabilityManagement.newEngineOnly }}
```

Comment:
it is, because  `newEngineOnly` is "included" in the new helper "nodeAnalyzer.deploImageAnalyzer" defined here:https://github.com/sysdiglabs/charts/pull/595/files#diff-c0c8ff035f55b2bed4c0503bbcb2d06cd9cc71064dee11c6ccd412dc0dcb3fafR251